setup.py

```diff
@@ -7,6 +7,8 @@
 import warnings
 import os
 
+PYTORCH_HOME = os.path.abspath(os.environ['PYTORCH_HOME']) if 'PYTORCH_HOME' in os.environ else None
```

Comment:
IMHO, making sure this `PYTORCH_HOME` has the two headers below that are included in the .cu file:
torch/csrc/jit/codegen/cuda/kernel_cache.h, and torch/csrc/jit/codegen/cuda/ops/all_ops.h?

what do you think?

Reply:
+1 Sounds like a reasonable check before attempting to build